samples/pom.xml

```diff
@@ -64,11 +64,20 @@
           <dependency>
             <groupId>com.github.tomakehurst</groupId>
             <artifactId>wiremock</artifactId>
-            <version>2.25.0</version>
+            <version>2.27.2</version>
           </dependency>
         </dependencies>
       </plugin>  
     </plugins>
+
+    <pluginManagement>
```

Comment:
Can we not manage the version of the site plugin for all children in the parent pom, rather than having to repeat this in each child?

Reply:
Yes all 4 can go now